src/base/path.h

```diff
@@ -51,6 +51,11 @@ class Path final
     bool isAbsolute() const;
     bool isRelative() const;
 
+#if defined(Q_OS_WIN)
+    bool isUNCPath() const;
+    QString getUNCRootPathStr() const;
+#endif
```

Comment:
Please move them to the bottom of `public:` section. To right before `private:` section.